### ISSUE TITLE
[BUG] HTML output doesn't work well in mobile

### ISSUE THREAD

**Issue by jimlyas** [Kotlin/dokka#2347]
**Describe the bug**
When opened from mobile devices, the page have features like when opened through desktop.

The other thing is it doesn't have toggle to switch between light and dark theme like desktop when opened with mobile.

**Expected behaviour**
The page doesn't have dark/light theme switches, and the menu just plain white

**Screenshots**
![image](https://user-images.githubusercontent.com/33854739/152741367-2a2d5efe-dec5-4d5c-8999-a2adcae0b8cd.png)
![image](https://user-images.githubusercontent.com/33854739/152741379-9bfb2067-59d9-415d-a705-4f4fb1b000c7.png)

**To Reproduce**
Just build your applicatioin using `dokkaHtmlMultiModule` and display it with mobile devices

**Dokka configuration**
Configuration of dokka used to reproduce the bug

```Kotlin
tasks.withType(DokkaMultiModuleTask::class.java).configureEach {
        moduleName.set("DevBase")
        outputDirectory.set(file("$rootDir/docs/src/reference"))
        pluginConfiguration<DokkaBase, DokkaBaseConfiguration> {
            footerMessage = "© 2021 Telkom DEV"
            customAssets = listOf(file("logo-icon.svg"))
        }
}
```

**Installation**
- Operating system: Windows
- Build tool: Gradle v7.0.2
- Dokka version: 1.6.10

**Additional context**
Please support GFM format too, I see many improvement in HTML format but not in GFM. Thanks

**Are you willing to provide a PR?**
No.

**Comment by robstoll** [comment]
workaround for menu on mobile:
```
/* fix for dokka, wrong colors in dark-mode */
@media screen and (max-width: 759px) {
    :root.theme-dark #container #leftColumn {
        background: var(--background-color);
    }
    :root.theme-dark #leftToggler {
        color: var(--default-font-color);
    }
}
```

**Comment by IgnatBeresnev** [comment]
The problem with the menu is fixed in #2836, and it's going to be released in 1.9.0

I've created a separate issue for the missing theme switcher button: #3000
